CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ This document describes changes between each past release.
   intended to simplify the experience of people who are new to Kinto.
   Addresses #687.
 - Removed ``initialize_cliquet()``, which has been deprecated for a while.
+- Removed ``cliquet_protocol_version``. Kinto already defines API
+  variations as part of its URL format (e.g. ``/v0``,
+  ``/v1``). Services based on kinto.core are free to use
+  ``http_api_version`` to indicate any additional changes to their
+  APIs.
```

Comment:
I think we should keep the http_protocol_version and remove the cliquet_protocol one